core/src/main/java/org/elasticsearch/action/search/MultiSearchRequest.java

```diff
@@ -38,6 +38,7 @@
  */
 public class MultiSearchRequest extends ActionRequest<MultiSearchRequest> implements CompositeIndicesRequest {
 
+    private int maxParallelSearchRequests = 10;
```

Comment:
Is this a good default? I was also thinking about taking into account the number of active copies. (by picking the the index with lowest number of copies and multiply that by a constant factor)